### PR TITLE
concent acceptance test: force accept after force download

### PR DESCRIPTION
**Pull request by etam**
Related to #4217

### CODE REVIEW

File: scripts/concent_acceptance_tests/force_accept/test_requestor_doesnt_send.py [hunk 1]
```diff
@@ -206,6 +214,7 @@ def test_requestor_responds_with_accept(self):
                 concent_public_key=self.variant['pubkey'],
             ),
         )
+        assert self.provider_fsr.subtask_id == fsr.subtask_id
```
Comment (jiivan) [inline scripts/concent_acceptance_tests/force_accept/test_requestor_doesnt_send.py]: self.assertEqual

File: scripts/concent_acceptance_tests/force_download/test_filetransfers.py [hunk 2]
```diff
@@ -24,9 +23,7 @@
 logger = logging.getLogger(__name__)
 
 
-class ForceGetTaskResultFiletransferTest(ForceDownloadBaseTest,
-                                         unittest.TestCase):
-
+class TestBase(ForceDownloadBaseTest):
```
Comment (jiivan) [inline scripts/concent_acceptance_tests/force_download/test_filetransfers.py:26]: This looks unrelated

Comment (etam) [reply scripts/concent_acceptance_tests/force_download/test_filetransfers.py:26]: what?

File: scripts/concent_acceptance_tests/force_accept/test_requestor_doesnt_send.py (hunk 1 above)
Comment (etam) [reply scripts/concent_acceptance_tests/force_accept/test_requestor_doesnt_send.py]: oh come on

**Review by shadeofblue — APPROVED** [review]
thank_you_roadsign

File: scripts/concent_acceptance_tests/force_download/test_filetransfers.py (hunk 2 above)
Comment (etam) [reply scripts/concent_acceptance_tests/force_download/test_filetransfers.py:26]: This `TestBase` is used in `force_accept_after_force_download.py`, if that's what you ask for.

**Review by jiivan — APPROVED** [review]
